### PR TITLE
Clean up cruft in useQueries

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: modules/useQueries.js [hunk 1]
```diff
@@ -41,7 +27,7 @@ const useQueries = (createHistory) =>
         return location
 
       if (location.query == null)
-        location.query = parseQueryString(location.search.substring(1))
+        location.query = parseQueryString(location.search)
```
Comment (taion) [inline modules/useQueries.js]: `query-string` handles leading `?`. This is breaking for legacy `qs` users, but I don't think we need to make that accommodation here.

**Comment by mjackson** [comment]
Can we get a test around this? Seems odd that none of our tests are breaking, no?

**Comment by taion** [comment]
Around what specifically? Query decoding from search is already covered here. It's intentional that this is non-breaking – that `substring(1)` operation is no-op when using `query-string`.

**Comment by taion** [comment]
I changed my mind. The substring change accomplishes nothing useful; all it does is make life harder for users of `qs`. I've updated this PR to just take out the old warning for nested objects.

**Pull request merged**